optuna_dashboard/ts/action.ts

```diff
@@ -265,6 +266,34 @@ export const actionCreator = () => {
       })
   }
 
+  const tellTrial = (
+    studyId: number,
+    trialId: number,
+    state: TrialState,
+    values?: string[]
+  ) => {
+    const message =
+      values === undefined
+        ? `id=${trialId}, state=${state}`
+        : `id=${trialId}, state=${state}, values=${values}`
+    tellTrialAPI(studyId, trialId, state, values)
+      .then(() => {
+        enqueueSnackbar(`Success to update trial (${message})`, {
+          variant: "success",
+        })
```

Comment:
Could you update the Recoil's `studyDetailsState` here like `saveStudyNote` action does?
https://github.com/optuna/optuna-dashboard/blob/29a36bdb6affbb3b065b4acf588f0fee45acd779/optuna_dashboard/ts/action.ts#L196-L205

Reply:
Fixed.